targets/XPlatCppSdk/source/code/source/playfab/OneDSHttpPlugin.cpp

```diff
@@ -3,8 +3,7 @@
 #ifndef DISABLE_ONEDS_API
 
 #include <playfab/OneDSHttpPlugin.h>
-
-#include <playfab/PlayFabTransportHeaders.h>
+#include <curl/curl.h>
```

Comment:
Do we want to keep all transport related includes in the PlayFabTransportHeaders.h? That was the original design of adding the header.

Reply:
No, we do not want to have a common shared header that includes the whole world (even separated by platform macros). Specific dependencies must stay only within the CPP code of particular plugins if they are not exposed publicly through their headers.

---
In reply to: [244868921](https://github.com/PlayFab/SDKGenerator/pull/368#discussion_r244868921) [](ancestors = 244868921)